### DISCUSSION TITLE
Unable to upload a file

### DISCUSSION THREAD

**Discussion by Korgan2004** [community/community#140437] [GitHub Education]
### Hi everyone,

Hi everyone, I want to apply for a student pack, but I can't because it requires to take a photo on the go, I can't upload it from my photo library or PDF. But it's okay, I did tried to upload it, but then it's all blurry and I can even see pixels on my photo. I thought I will use OBS camera with my document as a camera to make it more higher quality but it's the same blurry and pixely image. But it's not even half the problem... When I'm trying to apply it says "why are you not on campus", but why am I supposed to apply for it in my university where i'm suppose to study, why I can't do it home? And I'm not like in other town, I'm literally in the same town where my university at, just 4km away. And yes, I did read an article about common reasons of rejections, it's not relevant to me. Why I cant't upload my proof, It's just makes it very hard for everyone. I know that GitHub is trying to deal with dishonest people out here, but maybe yall should just check applications more thoroughly and better and there would be no problem with that.

**Comment by BDadmehr0** [comment]
It sounds really frustrating to deal with those application issues! Having to take a photo on the go can definitely lead to problems with quality, especially if you’re using a webcam or mobile setup. It’s odd that they require you to be on campus for something that could easily be done remotely, especially if you’re so close by.

It might be worth reaching out to the support team or student services at your university for help. They might have specific guidelines or alternative solutions for students facing these technical hurdles. Additionally, they could potentially offer a way to verify your identity that doesn’t require you to be physically present on campus.

For the photo quality issue, if you haven’t already, try adjusting the lighting or the angle of the camera, or use a higher resolution setting if available. Sometimes, a simple background change can also help.

**Reply by Korgan2004** [reply]
I tried, I have an iPhone, so the quality of the document will be okay. I even set up ma phone on a tripod, and even then it’s all blurry and pixely. And the lighting is okay. Why I can’t upload my proof, that will be much easier. And my university has a website that has an option to check if proof is eligible or not. My proof is written in a Russian language, but I read that GitHub can translate it by themselves, Russian is very popular language, so I thought that there will be no problem with that.

**Reply by BDadmehr0** [reply]
It sounds really frustrating to deal with these issues, especially when you're doing everything right on your end. Here are a few more thoughts that might help:

1. **iPhone Camera Settings**: Sometimes, the camera settings can affect photo quality. Make sure you’re using the highest resolution settings and that you're not in a low-light mode or a setting that might reduce clarity.

2. **Try Different Backgrounds**: If you haven't already, try taking the photo against a plain, contrasting background to help the document stand out more clearly.

3. **Proof Translation**: Since your proof is in Russian, it’s possible that the application system might not accept documents in languages other than English. You might want to check if they have specific guidelines for non-English documents.

4. **University Support**: If you haven't already, reach out to the university's support team or the specific department handling the application. Explain your situation about the language of your document and the upload issues. They might be able to provide you with a workaround or clarify their requirements.

5. **Alternative Submission Options**: Some universities allow for email submissions or alternative methods for documents if you explain your situation. It’s worth asking if this is an option.

It’s definitely challenging, but keeping communication open with your university can sometimes lead to solutions you might not have considered. Good luck, and I hope it gets resolved soon!
